linera-views/src/rocks_db.rs

```diff
@@ -72,8 +72,12 @@ impl KeyValueStore for RocksDbStoreInternal {
     async fn contains_key(&self, key: &[u8]) -> Result<bool, RocksDbContextError> {
         ensure!(key.len() <= MAX_KEY_SIZE, RocksDbContextError::KeyTooLong);
         let client = self.clone();
-        let key = key.to_vec();
-        Ok(tokio::task::spawn_blocking(move || client.db.key_may_exist(&key)).await?)
+        let key_copy = key.to_vec();
+        let test = tokio::task::spawn_blocking(move || client.db.key_may_exist(&key_copy)).await?;
```

Comment:
Maybe `let key_may_exist = { let key = key.to_vec(); tokio::task::spawn_blocking(...).await? };` ?

Reply:
Yes, that is nicer. Thank you.